src/UI/Art/SpriteArtist/SpriteMapView.java

```diff
@@ -320,16 +298,77 @@ else if( getCurrentGameMenu() == null )
       {
         menuArtist.drawMenu(mapGraphics, mapViewX, mapViewY);
       }
+      
+      // When we draw the map, we want to center it if it's smaller than the view dimensions
```

Comment:
Ah, and this is why you are drawing the background behind the map. At least on Linux, this results in white areas below and to the right of the map, covering the background.
I guess you stopped the size-snapping behavior in case the user expands the info menu? Hrm. I think I might prefer to just ensure the window is usable at the smallest size, and scales nicely if it gets expanded.
Failing that... maybe just ditch the animated background behind the map anyway, and use a nice neutral tone so there isn't non-game movement going on?

Reply:
I don't have super strong opinions on this,
You'll note my TODO about making the thing scroll freely, so it's viable at any size.

I personally liked the size-snapping, but it wouldn't work for the current iteration of the info menu and it bothered some other folks.
That being said, I think letting the UI hang out outside of the map when applicable is good too, to let you see the whole thing at once.

Reply:
Well, I'm certainly willing to entertain feedback from beta-testers, though I'm curious that anyone would want a window size bigger than the map. Was there any more specific feedback? If it's just that the info pages don't look right at the standard size, that's fixable.

Reply:
Well, for one thing, the map selection screen quickly gets to have too many to fit on screen at once. (though the real solution there is probably to add proper scrolling) (also I should get around to making it support subfolders)
They've been adding competitive maps for playtests, and there are a lot of those.

Showed it to a guy and he was visibly put out by the game not being able to take up his whole screen.
I think the next step might be to change the drawscale automagically when you resize the thing big enough, but that has its own possible issues.

Reply:
> he was visibly put out by the game not being able to take up his whole screen.

Has he... played Advance Wars? I guess it does take up the whole screen on a DS.

> change the drawscale automagically

I'm not really a fan of this idea... DrawScale is already its own option, and I don't want to automatically override that for the user.

Reply:
> Has he... played Advance Wars? I guess it does take up the whole screen on a DS.

Nope.

> I'm not really a fan of this idea... DrawScale is already its own option, and I don't want to automatically override that for the user.

Yeah, that's fair. It'd be hard to know when to *down*scale it too, and one-way automatic transitions are horrible UX.